# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/bacterium/bacterium_gui.diff
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-11-09 13:50:16.061560752 +0100
+++ ./Makefile	2003-11-09 13:51:45.827914192 +0100
@@ -7,6 +7,10 @@
 
 all: bacterium_ffd bacterium_gui
 
+# place -lncurses after bacterium_ffd.c
+bacterium_ffd: bacterium_ffd.c
+	$(CC) -Wall bacterium_ffd.c -lncurses -o bacterium_ffd
+
 bacterium_gui: bacterium_gui.sh
 	sed 's,./\(bacterium_ffd\),\1,' < bacterium_gui.sh > bacterium_gui
 	chmod +x bacterium_gui
